pkg/scheduler/api/constants.go

```diff
@@ -0,0 +1,6 @@
+package api
+
+const (
+	// EnablePredicateErrCacheKey is the key whether predicate error cache is enabled
+	EnablePredicateErrCacheKey = "predicateErrorCacheEnable"
```

Comment:
A little doubt, seems it's just a param and not an API, can we put it in another place?

Reply:
+1

Reply:
how about put them in `pkg/scheduler/conf/scheduler_conf.go`?

Reply:
I think this will be fine